draft-ietf-sidrops-rpki-prefixlist.xml

```diff
@@ -193,21 +216,85 @@
           The IP Address Delegation Extension <xref target="RFC3779"/> is not used in Signed Prefix List, and <bcp14>MUST NOT</bcp14> be present in the EE certificate.
         </li>
       </ol>
-    </section>
-
-    <section>
+      <section title="Validated Signed PrefixList Payload" anchor="VSP">
+      <t>
+        Normally there would be a single valid Signed Prefix List object for a given asID (see <xref target="operations"/>).
+        However, multiple valid Signed Prefix List objects which contain the same asID could exist.
+        In such cases, the union of the resulting address prefix members forms the complete set of members.
+        The complete set (resulting from a single or multiple PrefixLists) is locally stored by a relying party, and such an object is called the Validated SignedPrefixList Payload (VSP) for the asID in consideration.
+        For a given asID, if there is only a valid Signed Prefix List object with zero prefixes listed, then the VSP for the asID MUST show 'Empty' in place of the set of prefixes.
+      </t>
+      </section>
+      </section>
+    
+      <section title="Problems Solved by PrefixList" anchor="problems">
+      <t>
+        Given that a VSP exists for an asID, if a prefix route is announced whose origin AS matches the asID but the prefix is not listed in the VSP, then the route is considered Invalid per PrefixList verification.
+        The network operator may consider such a route ineligible <xref target="RFC4271"/> for route selection, regardless of the route's RPKI Route Origin Validation (ROV) status <xref target="RFC6811"/>.
+        Similarly, the operator may consider an ROV Invalid route ineligible for route selection, regardless of the route's PrefixList verification status.
+        If the above stated policy is implemented, PrefixList verification solves the additional routing security problems listed below while ROV detects accidental prefix hijacks.
+        More details concerning the procedures for PrefixList verification will be provided in a separate draft.
+      </t>
+      <ol>
+        <li>
+        <strong>AS Abuse While Hijacking an ROV-Unknown Prefix:</strong>
+        If AS A creates a PrefixList, it is protected in case an offending AS X inserts AS A as the origin AS and hijacks a third-party prefix not covered by a ROA.
+        </li>
+        <!--
+        The route will be ROV-Unknown and PrefixList Invalid and hence
+        considered ineligible by an RP (see #6 in <xref target="state-space"/>).
+        Thus, AS A protects itself from this form of AS abuse.
+        -->
+        <li>
+        <strong>AS Abuse Together with a Bogus ROA:</strong>
```

Comment:
The terminology "Bogus ROA" is not optimal, because the ROA is cryptographically valid, and the IP Prefix Resource holder is free to list any ASN they want in the origin field.

Reply:
“Non-reciprocal ROA” - because we can only guess about intent, and it might not even be malicious, but with SPL the RPs can conclude the Roa is not reciprocated. Thoughts?

Reply:
That seems like a more appropriate name. 

Proposed new text:

   2.  *AS Abuse Together with a Non-Reciprocal ROA:* 
        A non-reciprocal ROA is one which permits a prefix origination 
       from an AS but the AS has not included the prefix in its Signed Prefix List.
       If AS A creates a Signed Prefix List, it is protected in case an offending AS X
       inserts AS A as the origin AS and announces a prefix for which AS X has
       a non-reciprocal ROA showing AS A as the origin.

Reply:
@job 
Rethinking... we are describing an "attack type" here in which the offending AS X is engaged in a forged-origin prefix hijack (intentionally). So, it should be OK to use the term 'malicious ROA' which readers can readily understand.